lab_5_scrapper/scrapper.py

```diff
@@ -19,18 +75,55 @@ def __init__(self, path_to_config: pathlib.Path) -> None:
             path_to_config (pathlib.Path): Path to configuration.
         """
 
+        self.path_to_config = path_to_config
+        self._validate_config_content()
+        self.config = self._extract_config_content()
```

Comment:
Why store both instance and all its properties as attributes? Isn't it redundant?